source/agora/node/Ledger.d

```diff
@@ -202,6 +202,34 @@ public class Ledger
 
         return null;
     }
+
+
+    /***************************************************************************
+
+        Get the array of hashs the merkle branch.
+
+        Params:
+            block_height = the starting block height to begin retrieval from
```

Comment:
Wrong documentation, it's for `getBlocksFrom`

Reply:
Okay, I'll fix it.